build.gradle.kts

```diff
@@ -19,44 +22,49 @@ plugins {
 
 allprojects {
   apply(plugin = "spinnaker.base-project")
+  if (property("enablePublishing") == "true") {
+    apply(plugin = "spinnaker.project")
+  }
 
   group = "com.netflix.spinnaker.keel"
 }
 
 subprojects {
-  apply(plugin = "nebula.kotlin")
-
-  repositories {
-    jcenter()
-  }
+  if (name != "keel-bom") {
+    apply(plugin = "nebula.kotlin")
 
-  dependencies {
-    "implementation"(platform("com.netflix.spinnaker.kork:kork-bom:${property("korkVersion")}"))
+    repositories {
+      jcenter()
+    }
 
-    "implementation"("org.slf4j:slf4j-api")
+    dependencies {
+      "implementation"(platform("com.netflix.spinnaker.kork:kork-bom:${property("korkVersion")}"))
```

Comment:
Version has to be specified here or it's missing in the generated bom and then Gradle can't parse it.